### DISCUSSION TITLE
1.0.0 - More stability

### DISCUSSION THREAD

**Discussion by XorTroll** [XorTroll/Goldleaf#686] [General]
# Changelog

## General

- Some minor rewrites (in a probably unnoticeable way) improved the internals of the project

- As always, all libraries were updated to their latest versions

## Goldleaf

- NSP installs have been (finally) made multithreaded - I've personally noticed up to 25% faster install times, but this speedups might be even greater ;)

- Proper support for multi-program titles was added (for installing, browsing and exporting them correctly)

- Switched to an alternative result system via *arc*, so that now more system results have proper names/descriptions for more readability

- Improved various internal parts via caching or general optimizations

- Added support to check titles with no data (archived content, gamecard titles without the gamecard inserted, actually corrupted content, etc.)

- Extended keyboard support, now supporting other kinds of characters (particularly fixed support for account renaming)

- Fixed an annoying ticket/cert exporting bug which caused invalid NSP exports being generated

- Now one can mount and explore homebrew NROs' RomFs

- Updated some language translations (thanks a lot to all the people who frequently help with that <3)

- Added support for 12h/24h time display and configuration

- Support for NXThemes was removed

- Added a setting to allow custom NSP installation copy buffer sizes

- Slightly improved random UI color generation

- Fixed menu scrollbars behaving incorrectly in menus with too many items

- Fixed a bug where one could exit Goldleaf in the middle of installing NSPs, copying files, etc.

- Improved the file/directory copy menu and content export menu UIs

- Added a setting to disable showing hidden files/directories (those starting by '.' in their name)

- Now showing a deletion dialog after a NSP install is disabled by default, and a setting was added to enable/disable it

## Quark

- Fixed a bug where, when successfully reconnecting to Goldleaf after a connection loss, Quark would regardless close

<hr /><em>This discussion was created from the release <a href='https://github.com/XorTroll/Goldleaf/releases/tag/1.0.0'>1.0.0 - More stability</a>.</em>

**Comment by kavid** [comment]
update zh-Hans.json
[zh-Hans.zip](https://github.com/XorTroll/Goldleaf/files/12387989/zh-Hans.zip)

**Reply by XorTroll** [reply]
Added them, thanks!

**Comment by himo45** [comment]
Is there a way to change the game image?

**Comment by Ammako** [comment]
Not a bug new to 1.0.0, but changing settings in-app breaks custom color scheme, because background color doesnt get written to the updated config. Been that way since probably fdc9fe9
